lib/plugins/haskell.js

```diff
@@ -0,0 +1,28 @@
+import { HASKELL_IMPORT } from '../../packages/helper-grammar-regex-collection/index.js';
+import insertLink from '../insert-link';
+import preset from '../pattern-preset';
+
+export default class Haskell {
+  static resolve({ path, target }) {
+    const [, user, repo] = path.split('/');
+    const filePath = target.replace(/\./g, '/');
+    return [
+      `{BASE_URL}/${user}/${repo}/blob/master/src/${filePath}.hs`,
+      `{BASE_URL}/${user}/${repo}/blob/master/lib/${filePath}.hs`,
+      `{BASE_URL}/${user}/${repo}/blob/master/${filePath}.hs`,
```

Comment:
Are these file path conventions pretty strong in the Haskell community? I'm mostly just curious (OctoLinker uses heuristics in [other places](https://github.com/OctoLinker/browser-extension/issues/300), so it shouldn't be a problem here).